packages/git/src/browser/blame/blame-decorator.ts

```diff
@@ -200,23 +186,41 @@ export class BlameDecorator implements monaco.languages.HoverProvider {
 
 export namespace BlameDecorator {
 
-    export const maxWidth = 50; // character
+    export const defaultGutterBeforefStyles = <CSSStyleDeclaration>{
+        verticalAlign: 'bottom',
+        width: '36ch',
+        color: 'var(--theia-gitlens-gutterForegroundColor)',
+        backgroundColor: 'var(--theia-gitlens-gutterBackgroundColor)',
+        height: '100%',
+        display: 'inline-block',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis'
+    };
 
-    export const defaultGutterStyles = <CSSStyleDeclaration>{
-        width: `${maxWidth}ch`,
+    export const defaultGutterAfterStyles = <CSSStyleDeclaration>{
```

Comment:
Removing exports from this namespace (defaultGutterStyles, maxWidth, continuationStyle, highlightStyle) is a breaking change. Please add an entry to the changelog reflecting the changes.